### ISSUE TITLE
Some bug on ventrua 13.3

### ISSUE THREAD

**Issue by 13months** [osy/HaC-Mini#1073]
<img width="756" alt="花屏-1" src="https://user-images.githubusercontent.com/19258580/228160644-1dc5acc3-4275-45b9-8d12-28fb81f2cb5f.png">
<img width="788" alt="花屏-2" src="https://user-images.githubusercontent.com/19258580/228160659-14bb93bc-7570-4d9e-91e2-bb7f7293fa33.png">

Today update to ventura 13.3
and some bug display wrong

call for your help ，thanks

**Comment by SpookHCK** [comment]
Has anyone tried updating the lilu driver to 1.6.4?

**Comment by Needham73** [comment]
Yes. Still the same

**Comment by SpookHCK** [comment]
This reminds me of this [bug 316](https://github.com/osy/HaC-Mini/issues/316) from a long time ago... maybe that driver needs more work? wait for OSY to look at this...

**Comment by alenzhong** [comment]
I have the same issues,after various attempts, finally restored to macOS 13.2.1

**Comment by S1999U2** [comment]
> I have the same issues,after various attempts, finally restored to macOS 13.2.1

How did you do that? 
Afaik the only way is to completely reinstall macOS and then use Migration Assisstant to copy data from a backup.

Rolling back to 13.2.1 using Time Machine doesn't work

**Comment by SashonIT** [comment]
I also read a lot about Grafics Issues on other Systems....

Have a look here ---> https://www.hackintosh-forum.de/forum/thread/56383-macos-13-ventura-beta/?postID=771039#post771039

**Comment by SashonIT** [comment]
Maybe it has to do with the Fake ID of the Graphics Card.

Also This Problem was in the Beginning of Hackintoshing this Maschine with Mojave.  There it wqs more worse.
https://osy.gitbook.io/hac-mini-guide/details/metal-driver-fix

Then Big Sur ....

And now Ventura 13.3

Apple Changed something with Graphical Driver .

**Comment by monkeybape** [comment]
same problem

**Comment by mheyl** [comment]
Same experience here - although it's quite interesting.. - I have graphics issue in Safari, but not in Opera.. ; the main menue when hitting the Apps Icon from the dock is broken - the sub-menues are not... - weird...

- so I can live with it I guess (although I just upgraded to 13.3 and haven't looked at all apps yet..);

**Comment by SashonIT** [comment]
> Same experience here - although it's quite interesting.. - I have graphics issue in Safari, but not in Opera.. ; the main menue when hitting the Apps Icon from the dock is broken - the sub-menues are not... - weird...
> 
> * so I can live with it I guess (although I just upgraded to 13.3 and haven't looked at all apps yet..);

it’s in all apps !

i checked this

**Comment by x64code** [comment]
This makes some apps unusable:
Excel
Warp Terminal
Safari

Luckily VScode is working fine

**Comment by goodbest** [comment]
[Polaris22Fixup.kext.zip](https://github.com/osy/HaC-Mini/files/11155961/Polaris22Fixup.kext.zip)

@monkeybape @x64code @13months @S1999U2 @SpookHCK 
Can you try this kext in Ventura 13.3?
It's working for me.

**Comment by SashonIT** [comment]
> [Polaris22Fixup.kext.zip](https://github.com/osy/HaC-Mini/files/11155961/Polaris22Fixup.kext.zip)
> 
> @monkeybape @x64code @13months @S1999U2 @SpookHCK Can you try this kext in Ventura 13.3? It's working for me.

that sounds very auspicious !
I will try it these days and will reporting here.

Big Thanks forwarded to @goodbest

**Comment by KevisYuan** [comment]
> > [Polaris22Fixup.kext.zip](https://github.com/osy/HaC-Mini/files/11155961/Polaris22Fixup.kext.zip)
> > @monkeybape @x64code @13months @S1999U2 @SpookHCK Can you try this kext in Ventura 13.3? It's working for me.
> 
> great!!work for me now!!

I try install this kext but still have this problem ☹️

![image](https://user-images.githubusercontent.com/23292381/230059831-ea6cbb6b-4f06-47e1-8d24-9b12876f5b5f.png)

**Comment by SashonIT** [comment]
@KevisYuan which OpenCore Version you are Using?

Please update to the very latest Version 0.9.1. which was released yesterday.

Also update all very Important Kext's like Lilu,Whatevergreen and some other to the latest Version.

**Comment by KevisYuan** [comment]
> @KevisYuan which OpenCore Version you are Using?
> 
> Please update to the very latest Version 0.9.1. which was released yesterday.
> 
> Also update all very Important Kext's like Lilu,Whatevergreen and some other to the latest Version.

I solve it at last, thx a lot!

**Comment by ZhibinLiIntel** [comment]
> [Polaris22Fixup.kext.zip](https://github.com/osy/HaC-Mini/files/11155961/Polaris22Fixup.kext.zip)
> 
> @monkeybape @x64code @13months @S1999U2 @SpookHCK Can you try this kext in Ventura 13.3? It's working for me.

Do we need any new release in this repo?

**Comment by gconaty** [comment]
I'd certainly appreciate an 'official' release or less preferred a doc on how to update all the kexts manually.
Thanks for the fix.  Are there details on what was changed to make it work?

**Comment by yaoshengyi** [comment]
> I'd certainly appreciate an 'official' release or less preferred a doc on how to update all the kexts manually. Thanks for the fix. Are there details on what was changed to make it work?

Download source code, install package.pkg,run "build.sh" to  create directorys ，change source.txt in kext directory  you want to update.(update download link and md5) . then run build.sh again . The install pkg will be created in ./Installer/build/

**Comment by fishnetstalker** [comment]
Looks like this fixed graphics for me.

**Comment by gconaty** [comment]
> > I'd certainly appreciate an 'official' release or less preferred a doc on how to update all the kexts manually. Thanks for the fix. Are there details on what was changed to make it work?
> 
> Download source code, install package.pkg,run "build.sh" to create directorys ，change source.txt in kext directory you want to update.(update download link and md5) . then run build.sh again . The install pkg will be created in ./Installer/build/

Awesome and easy to build, just needed the correct links.   Thanks @goodbest  I was able to build from your branch.   Graphics are working again, thanks very much.

**Comment by monkeybape** [comment]
> Y
> 
> > [Polaris22Fixup.kext.zip](https://github.com/osy/HaC-Mini/files/11155961/Polaris22Fixup.kext.zip)
> > @monkeybape @x64code @13months @S1999U2 @SpookHCK Can you try this kext in Ventura 13.3? It's working for me.
> 
> You can also download this pkg for fresh install (github login is needed to download) https://github.com/osy/HaC-Mini/suites/12071453620/artifacts/635231939

there still have a problem.if i set lock screen any mins like 5mins.the screen will shutdown after 5mins,then wake up the device,the screen will screen flicker.

**Comment by SpookHCK** [comment]
> Y
> 
> > [Polaris22Fixup.kext.zip](https://github.com/osy/HaC-Mini/files/11155961/Polaris22Fixup.kext.zip)
> > @monkeybape @x64code @13months @S1999U2 @SpookHCK Can you try this kext in Ventura 13.3? It's working for me.
> 
> You can also download this pkg for fresh install (github login is needed to download) https://github.com/osy/HaC-Mini/suites/12071453620/artifacts/635231939

Works great; I installed and then upgraded to 13.3.... no problems so far! Thank you!

**Comment by HorstBrack75** [comment]
> [Polaris22Fixup.kext.zip](https://github.com/osy/HaC-Mini/files/11155961/Polaris22Fixup.kext.zip)
> 
> @monkeybape @x64code @13months @S1999U2 @SpookHCK Can you try this kext in Ventura 13.3? It's working for me.

It fixes the image artifacts. Thank you for!

On the other hand, the screen flickering issue seems to be back. Even with the flicker fix.
(1x4K, 1x1600x1200, both via DP)

**Comment by SpookHCK** [comment]
Still works fine after 13.3.1 upgrade. Thank you @goodbest

**Comment by fishnetstalker** [comment]
Update:  Major graphical problems have been resolved, but I still see distorted graphic for "preview" in mail app with PDF attachments. Also the "downloads" shortcut pop-up some graphics are distorted but not in finder app.

**Comment by SashonIT** [comment]
> Y
> 
> > [Polaris22Fixup.kext.zip](https://github.com/osy/HaC-Mini/files/11155961/Polaris22Fixup.kext.zip)
> > @monkeybape @x64code @13months @S1999U2 @SpookHCK Can you try this kext in Ventura 13.3? It's working for me.
> 
> You can also download this pkg for fresh install (github login is needed to download) https://github.com/osy/HaC-Mini/suites/12071453620/artifacts/635231939

Hello @goodbest 

First of all ..... a Big big Thank you for your PolarisFixup.kext Version 1.37

Second ....  a big big Thank for the re callibrated HacMini Installer.


I installed the 13.3.1 Clean on a second PArtition yesterday.  It runs through flawlessly without any Pain.

First : i made a 13.3.1 Install Stick.
Second: I installed the HacMini on the STick.

Third: i merched the Efi with my 13.2.1 EFI ---> Cause Of Thunderbolt 3 and DMAC / DMAR Table changes to run antelope Thunderbolt 3 Audio Interface.

And now its runnin Super Duper smooth and snappy.

Thank you again.

I wish you the very best... Goodbest
